cmake/sysbuild/image_signing_nrf54h20.cmake

```diff
@@ -7,9 +7,13 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
   # Build path to the sysbuild build directory.
   cmake_path(ABSOLUTE_PATH ZEPHYR_BINARY_DIR OUTPUT_VARIABLE app_binary_dir)
   cmake_path(GET app_binary_dir PARENT_PATH app_build_dir)
-  cmake_path(GET app_build_dir PARENT_PATH sysbuild_build_dir)
```

Comment:
There is a typo in the commit message ("chenge" should be "change") but only chenge this if the CI is read and you have to force push ;)